glide.yaml

```diff
@@ -38,3 +38,5 @@ import:
   version: release-3.11
 - package: github.com/openshift/client-go
   version: release-3.11
+- package: github.com/kubernetes-csi/external-snapshotter
+  version: ^v1.0.1
```

Comment:
May be wise to lock this at a specific version (no `^`)

Also no newline at the end of the file

Reply:
Done